packages/client/e2e/utils.ts

```diff
@@ -1,10 +1,17 @@
-import { expect, Locator } from "@playwright/test"
+import { expect, Locator, WorkerInfo } from "@playwright/test"
 
-export const OPENFIN_PROJECT_NAME = "openfin"
-export enum Timeout {
+export const IS_OPENFIN = (workerInfo: WorkerInfo): boolean => {
```

Comment:
call it `isOpenFin` .. only use that identifier syntax for constants (a carry-over from the likes of Java etc.)
minor .. can drop the { return ..}  (simplified arrow method syntax)

Reply:
thanks Alan, I've applied both changes 👍